Makefile

```diff
@@ -50,8 +54,12 @@ go-test: go-build
 test: bazel-test
 
 functest:
-	hack/dockerized "hack/build-func-tests.sh"
-	hack/functests.sh
+	if git diff --name-only | grep -q -v -e "md$$"; then \
```

Comment:
What if i want to run functoinal tests on top of master and the last merged commit happens to be a doc change? I think we should limit execution of tests only on CI, not per explicit request on make target.

Reply:
This may present a real problem in general, even on CI.

Jobs that may periodically want to run tests will encounter this issue.
I guess to resolve this we can provide dedicated targets to always run the tests no matter what.

But I am not favoring adding logic just for CI or local user, all options should exists for all in order to have consistency. The current difference between the CI and local run is unfortunate IMO.

Reply:
Right. Or just keep `functest` as it is and add this doc skip to a new smoke tests target you've mentioned.